src/avram/save_operation.cr

```diff
@@ -367,6 +367,10 @@ abstract class Avram::SaveOperation(T)
     {{ T.constant(:PRIMARY_KEY_NAME).id }}.value.nil?
   end
 
+  def changed_attributes
+    column_attributes.select(&.changed?)
+  end
```

Comment:
Looks to be unused. Could it be removed?

Reply:
Yep, remnant of precious experiments